src/uwtools/resources/rocoto.jsonschema

```diff
@@ -1,44 +1,55 @@
 {
-    "attrs": {
-        "realtime": {
+    "workflow": {
+        "attrs": {
```

Comment:
Does JSON Schema support making some of these attributes mandatory and others optional? And do we want to make the ones that are optional from Rocoto's perspective also optional in UW YAML? That second question isn't rhetorical -- it's a design decision potentially to-be-made. Looking at the Jinja2 template for the Rocoto XML doc, it looks like attributes will be created based on whatever is passed to Jinja2 to do the rendering, so I think we should define here anything that we want users to be able to use, and enforce the required/optional constraints we think appropriate.

Actually, given that, if I'm not mistaken, Jinja2 will roll out `key=value` attribute pairs based on whatever its given, probably we should not have empty keys in UW YAML just for the sake of schema conformance, as otherwise those will appear as `key=""` attributes, which Rocoto will probably reject. So, I think we'd want the Rocoto-optional attributes to be UW YAML-optional, too. I can't tell here if there's a required/optional distinction.